src/libraries/System.IO.Compression.ZipFile/tests/ZipFile.Encryption.cs

```diff
@@ -0,0 +1,371 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.IO.Compression.Tests
+{
+    public class ZipFile_EncryptionTests : ZipFileTestBase
+    {
+        public static IEnumerable<object[]> Get_SingleEntry_Data()
+        {
+            foreach (var method in new[]
+            {
+                ZipArchiveEntry.EncryptionMethod.ZipCrypto,
+                ZipArchiveEntry.EncryptionMethod.Aes128,
+                ZipArchiveEntry.EncryptionMethod.Aes192,
+                ZipArchiveEntry.EncryptionMethod.Aes256
+            })
+            {
+                yield return new object[] { method, false };
+                yield return new object[] { method, true };
+            }
+        }
+
+        public static IEnumerable<object[]> Get_MultipleEntries_SamePassword_Data()
+        {
+            foreach (var method in new[]
+            {
+                ZipArchiveEntry.EncryptionMethod.ZipCrypto,
+                ZipArchiveEntry.EncryptionMethod.Aes256
+            })
+            {
+                yield return new object[] { method, false };
+                yield return new object[] { method, true };
+            }
+        }
+
+        public static IEnumerable<object[]> Get_MultipleEntries_DifferentPasswords_Data()
+        {
+            foreach (var method in new[]
+            {
+                ZipArchiveEntry.EncryptionMethod.ZipCrypto,
+                ZipArchiveEntry.EncryptionMethod.Aes256
+            })
+            {
+                yield return new object[] { method, false };
+                yield return new object[] { method, true };
+            }
+        }
+
+        public static IEnumerable<object[]> Get_MixedPlainEncrypted_Data()
+        {
+            foreach (var method in new[]
+            {
+                ZipArchiveEntry.EncryptionMethod.ZipCrypto,
+                ZipArchiveEntry.EncryptionMethod.Aes128,
+                ZipArchiveEntry.EncryptionMethod.Aes256
+            })
+            {
+                yield return new object[] { method, false };
+                yield return new object[] { method, true };
+            }
+        }
+
+        public static IEnumerable<object[]> Get_Is_Async()
```

Comment:
There is `Get_Booleans_Data` defined in the base class that returns the same test data